reports/missedAutonaming.json

```diff
@@ -2771,6 +2771,65 @@
       }
     }
   },
+  "aws-native:datazone:GroupProfile": {
```

Comment:
👁️

Reply:
For these resources in particular, inferring something like userIdentifier from a Pulumi name seems undesirable, leaving as is.